Update Pond Strider, Flea, associated outfits (#4494)
endless-sky/endless-sky: data/wanderer/wanderers start.txt

		dialog "That was the last of the new Unfettered ships. You can now report back to Tele'ek on <planet>."
	on visit
		dialog `Tele'ek approaches you when you land. "I have [reports, word] that the Unfettered ships are still [present, alive] within Prakacha'a. Please destroy them all before returning."`

mission "Wanderers Solifuge Recon 3"
	landing
	name "New Unfettered ships"

> 	description "Ambassador Sayari is concerned that the development of the Solifuge could become a problem for the Hai, and has asked that you deliver a message to the council of elders on <destination>."

Reviewer: ```suggestion
	description "Ambassador Sayari is concerned that the development of the Solifuge could become a problem for the Hai and has asked that you deliver a message to the council of elders on <destination>."
```

	source "Vara Ke'sok"
	destination "Hai-home"
	clearance
	to offer
		not "Wanderers Evacuation 1: offered"
		or
			has "Wanderers Solifuge Recon 2: done"
			has "Wanderers Pond Strider Recon 2: done"
	on offer
		conversation
			`Tele'ek and Sayari listen carefully as you outline the strengths and weaknesses of the new Unfettered carriers. "I am not greatly [concerned, frightened] by tiny fighters," Tele'ek says. "The weapon that has done us most harm is their ion cannons, which no ship so small can [wield, carry]. But thank you for your assistance."`
			`	Sayari looks more concerned about this information. "These carriers may not be an issue for the Wanderers, but this development could cause trouble for the Hai. I have written a message describing the situation." She hands you a data chip. "Please deliver it to the council of elders on <planet>."`
			`	"Please be quick if you are to deliver this message now," Tele'ek says. "We have no other [fights, brawls] for you to take part in right now, but once you return here, you will find another evacuation fleet that needs an [escort, protector]."`
				accept

mission "Wanderers Solifuge Recon 4"
	landing
	name "Return to <planet>"

> 	description "Return to <destination> where an evacuation fleet is waiting to be escorted."

Reviewer: ```suggestion
	description "Return to <destination>, where an evacuation fleet is waiting to be escorted."
```

	source "Hai-home"
	destination "Vara Ke'sok"
	to offer
		has "Wanderers Solifuge Recon 3: done"
	on offer
		conversation
			`You give the message from Sayari to the spaceport authorities so that they can deliver it to the council of elders. After the message has been delivered, the council requests that you meet with them.`
			`	"Greetings again, <first> <last>," one of the elders says to you. "It is our understanding that our Unfettered brothers and sisters have created a carrier ship. How difficult was it for you to combat it?"`
			choice
				`	"The carrier is strong, but not strong enough, and can be easily taken down."`
				`	"It was easy for me, but I can't say it will be the same for your ships to combat."`

> 				`	"It was a difficult fight, as the carrier is very powerful."`

Reviewer: ```suggestion
				`	"It was a difficult fight. The carrier is very powerful."`
```

			`	The elders once again mumble among themselves. They continue to ask Torsa a number of questions about this new carrier. How difficult is one to produce? How much does it cost? What are the capabilities of the drones? How useful would it be without being able to repair the hull of its drones?`
			`	After having all their questions answered, the elders seem to be interested in the idea of Torsa's new ship. "This hull repair technology indeed sounds necessary," an elder says. "We will provide what funding we can to help fast track its development."`
			`	Torsa's father turns to you. "You have traveled many systems, <first> <last>. Perhaps you might be able to help. If you know of any peoples who have such technology from elsewhere in the galaxy, should it exist, then perhaps we could meet with them."`
			choice
				`	"I'm sorry, but I can't help right now. I need to return to the Wanderers."`
					goto busy
				`	"I'd be glad to help. I'm sure I could find someone with hull repair technology that could help."`
			

> 			`	"Sayari's message mentions that you are still needed by the Wanderers," Torsa's father responds. "Perhaps you should return to where you are needed the most, but when you are able to help us, we will gladly accept it." The elders all thank you for bringing Sayari's message and bid you a safe travel back to <planet>.`

Reviewer: ```suggestion
			`	"Sayari's message mentions that you are still needed by the Wanderers," Torsa's father responds. "Perhaps you should return to where you are needed the most, but when you are able to help us, we will gladly accept it." The elders all thank you for bringing Sayari's message and bid you safe travels back to <planet>.`
```

				accept
			
			label busy

> 			`	"This is understandable," Torsa's father responds. "Return to where you are needed, but when you are able to help, we will gladly accept it." The elders all thank you for bringing Sayari's message and bid you a safe travel back to <planet>.`

Reviewer: ```suggestion
			`	"This is understandable," Torsa's father responds. "Return to where you are needed, but when you are able to help, we will gladly accept it." The elders all thank you for bringing Sayari's message and bid you safe travels back to <planet>.`
```

		fail

mission "Wanderers Evacuation 1"
	landing
	name "Evacuate <planet>"
	description `Escort another fleet of Wanderer refugee transports to <destination>.`
	autosave
	source "Vara Ke'sok"
	destination "Vara Ke'stai"
	to offer
		has "Wanderers Solifuge Recon 4: done"
	on offer
		conversation

> 			`When you return to Wanderer space, you find four Wanderer ships that you've never seen before are parked near the spaceport. These must be the dedicated transport ships that Iktat Rek mentioned. One of the captains approaches you and says, "We are evacuating the worlds that will be [isolated, trapped] if the Unfettered gain more territory. Will you escort us to <planet>, and from there to the capital?"`

Reviewer 2: While this change does make this chunk flow better from a literary point of view, remember that the first part "When you return to Wanderer Space" isn't something that needs to be said because the player is *doing* that. You are basically describing what the player has already done. Essential for a book, but not important when the player has just done that action in-game. 

As such, I don't think this change is necessary or beneficial.

Reviewer 3: Well this form of transition is used elsewhere in the game, typically in the form `"When you return to <planet>"` or similar. As you say, it makes thing flow better from a literary standpoint, which is the whole reason to include a transition like this, bridging the gap between the gameplay side of things and the mission side of things.
